### PR TITLE
OCPBUGS#14864: Add known issue for Selinux relabeling issue for 4.13

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.13 only

Issue: https://issues.redhat.com/browse/OCPBUGS-14864

Link to docs preview: https://61561--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-known-issues

QE review:
- [x] QE has approved this change.

Additional information:

**PTAL**: @gcharot, @chao007, @sferich888, @xltian, @mwerner2113, @gnufied

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -2845,6 +2845,8 @@ This affects the achievable latency of such clusters and might prevent proper op
 
 * {product-title} 4.12 clusters on the Nutanix platform may have an `Upgradeable=False` condition if they are missing configuration needed for the Nutanix Cloud Control Manager (CCM). To resolve this condition, see: link:https://access.redhat.com/solutions/7014068[How to create the ConfigMaps and Secrets needed to upgrade to OpenShift 4.13 when using Nutanix as a Platform].
 
+* Currently, when using a persistent volume (PV) that contains a very high file count, the pod may not start or takes an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
```
Comment (maxwelldb) [inline release_notes/ocp-4-13-release-notes.adoc:2848]: ISG: Use "may" only to indicate permissions. 

```suggestion
* Currently, when using a persistent volume (PV) that contains a very high file count, the pod might not start or can an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
```

**Review by maxwelldb — COMMENTED** [review]
One suggestion.

Also, does this need to go through change management?

File: release_notes/ocp-4-13-release-notes.adoc (hunk 1 above)
Comment (gnufied) [reply release_notes/ocp-4-13-release-notes.adoc:2848]: same follow up comment as - https://github.com/openshift/openshift-docs/pull/61570#discussion_r1242490942

**Comment by gnufied** [comment]
lgtm

**Comment by gcharot** [comment]
lgtm

File: release_notes/ocp-4-13-release-notes.adoc (hunk 1 above)
Comment (lpettyjo) [reply release_notes/ocp-4-13-release-notes.adoc:2848]: Done.

**Comment by lpettyjo** [comment]
> One suggestion.
> 
> Also, does this need to go through change management?

Yes. And it is.

**Comment by xltian** [comment]
lgtm

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -2845,6 +2845,8 @@ This affects the achievable latency of such clusters and might prevent proper op
 
 * {product-title} 4.12 clusters on the Nutanix platform may have an `Upgradeable=False` condition if they are missing configuration needed for the Nutanix Cloud Control Manager (CCM). To resolve this condition, see: link:https://access.redhat.com/solutions/7014068[How to create the ConfigMaps and Secrets needed to upgrade to OpenShift 4.13 when using Nutanix as a Platform].
 
+* Currently, when using a persistent volume (PV) that contains a very large number of files, the pod might not start or can take an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. Also, for ODF users, see this link:https://access.redhat.com/solutions/6906261[workaround article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
```
Comment (gcharot) [inline release_notes/ocp-4-13-release-notes.adoc:2848]: The ODF known issues does not container the KB#6906261 so please remove in order to be aligned with their release notes. Thanks!

Comment (lpettyjo) [reply release_notes/ocp-4-13-release-notes.adoc:2848]: Done.

**Pull request merged**